### DISCUSSION TITLE
How do I place the switch on the right side

### DISCUSSION THREAD

**Discussion by yahiaboudah** [mantinedev/mantine#1536] [Q&A] [answered]
### What package has an issue

@mantine/core

### Describe the bug

I can't place the switch on the right side, there's no style prop

### In which browser did the problem occur

Chrome

### Do you know how to fix the issue

No

### Are you willing to participate in fixing this issue and create a pull request with the fix

Yes

### Possible fix

Don't know

**Comment by rtivital** [comment, marked as answer]
Place on the right side of what? Can you provide an example of what you are trying to achieve?

**Reply by yahiaboudah** [reply]
```
import { Switch } from '@mantine/core';

function Demo() {
  return (
    <Switch
      label="I agree to sell my privacy"
    />
  );
}
```

This puts the switch on the left side of the text, (SWITCH -- TEXT). There's no option that allows me to display it as (TEXT -- SWITCH)

**Reply by rtivital** [reply]
https://codesandbox.io/s/nostalgic-blackburn-kbom9v?file=/src/App.tsx

**Reply by yahiaboudah** [reply]
Beautiful, thanks a ton!
